pennathletics/scraper.py

```diff
@@ -0,0 +1,153 @@
+import requests
+from bs4 import BeautifulSoup
+import models
+import re
+
+SPORTS = {
+    'M_Baseball' : {
+        'SPID'  : 548,
+        'SPSID' : 8834
+    },
+    'M_Basketball' : {
+        'SPID'  : 539,
+        'SPSID' : 8627
+    },
+    'M_Track_Cross' : {
+        'SPID'  : 542,
+        'SPSID' : 8699
+    },
+    'M_Fencing' : {
+        'SPID'  : 607,
+        'SPSID' : 10603
+    },
+    'M_Football' : {
+        'SPID'  : 537,
+        'SPSID' : 8571
+    },
+    'M_Sprint_Football' : {
+        'SPID'  : 612,
+        'SPSID' : 10618
+    },
+    'M_Golf' : {
+        'SPID'  : 550,
+        'SPSID' : 8895
+    },
+    'M_Lacrosse' : {
+        'SPID'  : 544,
+        'SPSID' : 8745
+    },
+    'M_Heavy_Rowing' : {
+        'SPID'  : 610,
+        'SPSID' : 10659
+    },
+    'M_Light_Rowing' : {
+        'SPID'  : 613,
+        'SPSID' : 10647
+    },
+    'M_Soccer' : {
+        'SPID'  : 604,
+        'SPSID' : 10672
+    },
+    'M_Squash' : {
+        'SPID'  : 605,
+        'SPSID' : 10685
+    },
+    'M_Swimming' : {
+        'SPID'  : 611,
+        'SPSID' : 10698
+    },
+    'M_Tennis' : {
+        'SPID'  : 552,
+        'SPSID' : 8927
+    },
+    'M_Wrestling' : {
+        'SPID'  : 543,
+        'SPSID' : 8720
+    },
+    'W_Basketball' : {
+        'SPID'  : 540,
+        'SPSID' : 8650
+    },
+    'W_Track_Cross' : {
+        'SPID'  : 558,
+        'SPSID' : 10758
+    },
+    'W_Fencing' : {
+        'SPID'  : 608,
+        'SPSID' : 10715
+    },
+    'W_Hockey' : {
+        'SPID'  : 609,
+        'SPSID' : 10583
+    },
+    'W_Golf' : {
+        'SPID'  : 551,
+        'SPSID' : 8903
+    },
+    'W_Gymnastics' : {
+        'SPID'  : 545,
+        'SPSID' : 8763
+    },
+    'W_Lacrosse' : {
+        'SPID'  : 554,
+        'SPSID' : 8966
+    },
+    'W_Rowing' : {
+        'SPID'  : 555,
+        'SPSID' : 8993
+    },
+    'W_Soccer' : {
+        'SPID'  : 541,
+        'SPSID' : 8673
+    },
+    'W_Softball' : {
+        'SPID'  : 549,
+        'SPSID' : 8856
+    },
+    'W_Squash' : {
+        'SPID'  : 606,
+        'SPSID' : 10741
+    },
+    'W_Swimming' : {
+        'SPID'  : 546,
+        'SPSID' : 8790
+    },
+    'W_Tennis' : {
+        'SPID'  : 553,
+        'SPSID' : 8943
+    },
+    'W_Volleyball' : {
+        'SPID'  : 538,
+        'SPSID' : 8606
+    }
+}
```

Comment:
1. You should have this in a separate .py file for all the miscellaneous hardcoded data.
2. You should be using [namedtuples](https://docs.python.org/2/library/collections.html#collections.namedtuple), not dictionaries.